dcompose/dcompose.go

```diff
@@ -207,18 +209,27 @@ func (j *JobCompose) InitFromJob(job *model.Job, cfg *viper.Viper, workingdir st
 	}
 
 	// Add the final output job
+	excludesPath := path.Join(workingdir, UploadExcludesFilename)
+	excludesMount := path.Join(CONFIGDIR, UploadExcludesFilename)
+
+	uploadOutputsCmd := job.FinalOutputArguments()
+
+	uploadOutputsCmd = append(uploadOutputsCmd, "--exclude")
+	uploadOutputsCmd = append(uploadOutputsCmd, excludesMount)
```

Comment:
I feel like a more consistent way to do this block instead of like this would be to add an argument to `job.FinalOutputArguments` which is the path to the exclude file. Then, the code in `model` still handles all the creation of the command line and stuff. This is how e.g. the inputs (above) work, where they have `Command: input.Arguments(job.Submitter, job.FileMetadata)`, which in turn does the construction of arguments and such.